scripts/CEE/collect-verbose-ca-logs/script.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+set -e
+set -o nounset
+set -o pipefail
+
+# Set variables
+GREEN="\033[32m"
+RESET="\033[0m"
+NAMESPACE="openshift-machine-api"
+CA_NAME="default"
+
+# Get current logVerbosity value
+current_log_verbosity=$(oc get ca $CA_NAME -o jsonpath='{.spec.logVerbosity}')
+
+# Check if current_log_verbosity is empty
+if [ -z "$current_log_verbosity" ]; then
+  echo "Failed to retrieve current logVerbosity value."
+  exit 1
+fi
+
+echo
+echo "CURRENT LOG VERBOSITY = $current_log_verbosity"
+echo
+
+# Update logVerbosity to 6
+oc patch ca $CA_NAME --type='json' -p='[{"op": "replace", "path": "/spec/logVerbosity", "value": 6}]'
+
+updated_log_verbosity=$(oc get ca $CA_NAME -o jsonpath='{.spec.logVerbosity}')
+echo
+echo "UPDATED LOG VERBOSITY = $updated_log_verbosity"
+echo
+
+# Wait for the update to take effect
+echo "Waiting for the log verbosity to update..."
+
+# Sleep for 5 seconds so that the new pod name gets reflected
+sleep 5
+
+CA_POD=$(oc get pods -n openshift-machine-api | grep cluster-autoscaler-default | awk '{print $1}')
+echo "POD Name: $CA_POD"
+
+while true; do
+  # Get the name and status of the cluster-autoscaler pod
+  POD_STATUS=$(oc get pod "$CA_POD" -n $NAMESPACE -o jsonpath='{.status.phase}')
+
+  # Check if the pod status is "Running"
+  if [ "$POD_STATUS" == "Running" ]; then
+    echo "The cluster-autoscaler pod is now 'Running'."
+    break
+  fi
+
+  # Print the current status and wait before rechecking
+  echo "Current status: $POD_STATUS. Waiting..."
+  sleep 5
+done
+
+# Get the name of the Cluster Autoscaler pod
+CA_POD=$(oc get pods -n openshift-machine-api | grep cluster-autoscaler-default | awk '{print $1}')
```

Comment:
Do we need to get it again here?

Reply:
Not required as we are already reading it while implementing the check condition for nee pod's running state. I have removed #58 and #59 lines.